src/plugins/intel_cpu/tests/unit/shape_inference_test/multinomial_shape_inference_test.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multinomial_shape_inference.hpp"
+
+#include <gtest/gtest.h>
+
+#include "dimension_util.hpp"
+#include "utils.hpp"
+
+using namespace ov;
+using namespace ov::intel_cpu;
+
+TEST(StaticShapeInferenceTest, MultinomialDefaultShapeInferenceTest) {
+    auto param0 = std::make_shared<ov::op::v0::Parameter>(element::f32, PartialShape{4});
```

Comment:
Name op inputs in same way as in `type_prop` tests to give them some meaning.

Reply:
Applied!